docker-compose.yml

```diff
@@ -19,6 +30,9 @@ services:
                 sleep 10;
                 make airflow_init airflow_up;
             "
+        environment:
+            - AIRFLOW_CONN_SSH_RSERVER=ssh://rstudio:rstudio_password@rserver:22
+            - AIRFLOW_CONN_POSTGRES_DATALAKE=postgres://postgres:airflow@airflow:5432
```

Comment:
Same note as above:

The address provided here is pointing to the airflow metadata database,  the datalake database would look something like `postgres://postgres:airflow@airflow:5432` (or 5430, I am unsure, as Chow is using an alternative port that I am not familiar enough with docker to explain). Do you even need this connection in your environment?